azure/src/main/java/com/microsoft/azure/management/Azure.java

```diff
@@ -437,6 +439,7 @@ private Azure(RestClient restClient, String subscriptionId, String tenantId, Aut
         this.monitorManager = MonitorManager.authenticate(restClient, subscriptionId);
         this.eventHubManager = EventHubManager.authenticate(restClient, subscriptionId);
         this.subscriptionId = subscriptionId;
+        this.tenantId = tenantId;
```

Comment:
If `authenticated` is available, use `authenticated.tenantId()` would save this variable?

Reply:
Done